### PR TITLE
Htxfix

### PR DESCRIPTION
**Pull request by PraveenPenguin**
Fixed issues in htx_test 
1- fixed issue in run_smt subroutine as one process.system cmd not valid
2- change process.run('htxcmdline -geterrlog') to process.system('htxcmdline -geterrlog')
as if this command fails it make script fail and process.system is more suitable here
3- fix pep8 issue
move htx_test.data to htx_test.py.data that is more relevent.

Signed-off-by: Praveen K Pandey <praveen@linux.vnet.ibm.com>

### CODE REVIEW

File: generic/htx_test.py.data/htx_test.yaml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+time_limit: 2
```
Comment (narasimhan-v) [inline generic/htx_test.py.data/htx_test.yaml]: You should remove the `generic/htx_test.data/htx_test.yaml `

File: generic/htx_test.py [hunk 2]
```diff
@@ -56,8 +56,8 @@ def setUp(self):
             packages.extend(['gcc-c++', 'ncurses-devel',
                              'dapl-devel', 'libcxl-devel'])
         elif detected_distro.name == "Ubuntu":
-            packages.extend(['libncurses5', 'g++', 'libdapl-dev', 'ncurses-dev',
-                             'libncurses-dev', 'libcxl-dev'])
+            packages.extend(['libncurses5', 'g++', 'libdapl-dev',
+                             'ncurses-dev', 'libncurses-dev', 'libcxl-dev'])
```
Comment (narasimhan-v) [inline generic/htx_test.py]: pep8 and pylint fixes generally go as a separate commit, as per previous PRs. Consider that

Comment (PraveenPenguin) [reply generic/htx_test.py]: @narasimhan-v sure make sense

File: generic/htx_test.py.data/htx_test.yaml (hunk 1 above)
Comment (PraveenPenguin) [reply generic/htx_test.py.data/htx_test.yaml]: oh sorry forgot that will do , thanks for review

**Comment by PraveenPenguin** [comment]
@narasimhan-v adopted the changes , can you please have a look
